Framework/PythonInterface/plugins/algorithms/ComputeCalibrationCoefVan.py

```diff
@@ -54,9 +54,17 @@ def PyInit(self):
                              direction=Direction.Output),
                              ("Name the workspace that will contain the " +
                               "calibration coefficients"))
+        self.declareProperty("Temperature",
+                             defaultValue=Property.EMPTY_DBL,
+                             validator=FloatBoundedValidator(lower=0.0),
+                             direction=Direction.Input,
+                             doc=("Temperature during the experiment, in " +
+                                  "Kelvins."))
```

Comment:
Maybe it should be mentioned here (and in the `rst`) that this overrides the value in the sample log (and that leaving the field empty will use the sample log value)?